src/tools/info/tl_info.c

```diff
@@ -471,6 +471,63 @@ static void print_md_info(uct_component_h component,
     ;
 }
 
+static void print_cm_info(uct_component_h component,
+                          const uct_component_attr_t *component_attr,
+                          const char *cm_name, int print_opts,
+                          ucs_config_print_flags_t print_flags,
+                          const char *req_tl_name)
+{
+    ucs_status_t status;
+    uct_cm_config_t *cm_config;
+    uct_cm_attr_t cm_attr;
+    uct_cm_h cm;
+    ucs_async_context_t *async;
+    uct_worker_h worker;
+
+    status = ucs_async_context_create(UCS_ASYNC_MODE_THREAD_SPINLOCK, &async);
+    if (status != UCS_OK) {
+        goto out;
+    }
+
+    status = uct_worker_create(async, UCS_THREAD_MODE_SINGLE, &worker);
+    if (status != UCS_OK) {
+        goto out_async_ctx_destroy;
+    }
+
+    status = uct_cm_config_read(component, NULL, NULL, &cm_config);
+    if (status != UCS_OK) {
+        goto out_destroy_worker;
+    }
+
+    status = uct_cm_open(component, worker, cm_config, &cm);
+    uct_config_release(cm_config);
+    if (status != UCS_OK) {
+        printf("# < failed to open connection manager %s >\n", cm_name);
+        goto out_destroy_worker;
+    }
+
+    cm_attr.field_mask = UCT_CM_ATTR_FIELD_MAX_CONN_PRIV;
+    status = uct_cm_query(cm, &cm_attr);
+    if (status != UCS_OK) {
+        printf("# < failed to query connection manager >\n");
+        goto out_close_cm;
+    } else {
+        printf("#\n");
+        printf("# Connection manager: %s\n", cm_name);
+        printf("#     Component: %s\n", component_attr->name);
+        printf("#        max_conn_priv: %zu bytes\n", cm_attr.max_conn_priv);
```

Comment:
can we print devices and IP addresses (in separate PR)? I think it would be useful

Reply:
you mean all the available ips on the host?

Reply:
yes, assuming that sock cm is supported. 
Something similar to `ifconfig` but simpler and grouped by components